admin/tabs/activity/tracking/ActivityTrackingModel.ts

```diff
@@ -103,50 +102,35 @@ export class ActivityTrackingModel extends HoistModel {
             ] as CubeFieldSpec[]
         });
 
-        this.groupingChooserModel = new GroupingChooserModel({
-            dimensions: this.cube.dimensions,
-            persistWith: this.persistWith,
-            initialValue: this._defaultDims
-        });
-
         this.filterChooserModel = new FilterChooserModel({
-            bind: this.cube.store,
             fieldSpecs: [
-                'category',
-                'month',
-                'username',
-                'device',
-                'browser',
-                'msg',
-                'userAgent',
                 {
-                    field: 'elapsed',
-                    valueRenderer: v => {
-                        return fmtNumber(v, {
-                            label: 'ms',
-                            formatConfig: {thousandSeparated: false, mantissa: 0}
-                        });
-                    }
+                    field: 'category',
+                    valueRenderer: v => v,
+                    enableValues: true
                 },
                 {
-                    field: 'dateCreated',
-                    example: 'YYYY-MM-DD',
-                    valueParser: (v, op) => {
-                        let ret = moment(v, ['YYYY-MM-DD', 'YYYYMMDD'], true);
-                        if (!ret.isValid()) return null;
-
-                        // Note special handling for '>' & '<=' queries.
-                        if (['>', '<='].includes(op)) {
-                            ret = moment(ret).endOf('day');
-                        }
-
-                        return ret.toDate();
-                    },
-                    valueRenderer: v => fmtDate(v),
-                    ops: ['>', '>=', '<', '<=']
+                    field: 'username',
+                    valueRenderer: v => v,
```

Comment:
Don't think you need these valueRendererers - that's the default, only need this prop when customizing